app/controllers/concerns/on_board_single_sign_on_user.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class OnBoardSingleSignOnUser
+  include Interactor
+
+  after do
```

Comment:
TODO: Need to test or otherwise validate that this will NOT fire if the interactor fails